common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationRequest.java

```diff
@@ -292,7 +292,12 @@ public Uri getAuthorizationRequestAsHttpRequest() {
         final Uri.Builder uriBuilder = Uri.parse(getAuthorizationEndpoint()).buildUpon();
 
         for (Map.Entry<String, Object> entry : qpMap.entrySet()) {
-            uriBuilder.appendQueryParameter(entry.getKey(), entry.getValue().toString());
+            if (entry.getKey() != null && entry.getValue() != null) {
```

Comment:
we probably don't need null check for key here.

Reply:
yes agree, just added as it's no harm